server/integration/endpoint/src/main/java/org/infinispan/server/endpoint/EndpointLogger.java

```diff
@@ -184,5 +185,9 @@
    @LogMessage(level = INFO)
    @Message(id = 10032, value = "No Security realm declared for endpoint '%s'. Using default encryption realm.")
    void noSSLContextForSni(String endpoint);
+
+   @LogMessage(level = INFO)
+   @Message(id = 10033, value = "Starting server %s without transport (use multi tenant router instead)")
```

Comment:
No no, it's not a warning. It's just an information that the endpoint has started but you won't be able to access it using ordinary REST or Hot Rod Client. You need to use multi tenant mode (e.g. REST prefixes, SNI in Hot Rod client etc).